docs/io/ascii/fast_ascii_io.rst

```diff
@@ -144,7 +144,7 @@ relatively unimportant, this converter might be the best option in
 performance-critical scenarios.
 
 `Here
-<https://nbviewer.jupyter.org/github/astropy/astropy-notebooks/blob/master/io/ascii/conversion_profile.ipynb>`__
+<https://nbviewer.jupyter.org/github/astropy/astropy-notebooks/blob/main/io/ascii/conversion_profile.ipynb>`__
```

Comment:
Unrelated to this PR, but are these notebooks are not tested and in fact don't work any more, so better to remove them from the docs altogether.

Reply:
OK, let's continue discussion at #11381.

Reply:
This will also require re-naming that repo's branch, but I'll just do that right now since it's incredibly low traffic.

Reply:
Notebooks are removed. But if we want to keep that repo, should just rename the branch there anyway though that is out of scope of this PR now.